monkey/monkey_island/cc/services/authentication_service/flask_resources/token.py

```diff
@@ -0,0 +1,45 @@
+import logging
+from http import HTTPStatus
+
+from flask import request
+
+from monkey_island.cc.flask_utils import AbstractResource, responses
+
+from ..authentication_facade import AuthenticationFacade
+from .utils import ACCESS_TOKEN_KEY_NAME, REFRESH_TOKEN_KEY_NAME
+
+logger = logging.getLogger(__name__)
+
+
+class Token(AbstractResource):
+    """
+    A resource for refreshing tokens
+    """
+
+    urls = ["/api/token"]
```

Comment:
This name is pretty meaningless. It's also violates RESTful principles. If I POST to `/api/token`, I should be passing it some data to use to create a "token" resource. 

This is effectively a login/authentication call. It is not RESTful pretty much by definition; it's RPC. If it's an RPC call, it should have an RPC name.

Suggest: `POST /api/refresh-authentication-token`